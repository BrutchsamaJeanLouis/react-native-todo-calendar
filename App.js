import React from 'react'
import {StyleSheet, Text, View } from 'react-native'
import { Provider as StoreProvider, useSelector } from 'react-redux'
import { Provider as PaperProvider } from 'react-native-paper'
import { MD3LightTheme as DefaultTheme } from 'react-native-paper'
import { store } from './redux/store'
import HolidayList from './views/HolidayList'
import { styleConstants } from './constants/style-const'
import Index from './Index'

const theme = {
  ...DefaultTheme,
  colors: {
    ...DefaultTheme.colors,
    primary: '#0066ff',
    secondary: 'yellow'
  }
}

export default function App () {
  return (
    <StoreProvider store={store}>
      <PaperProvider theme={theme}>
        <Index />
      </PaperProvider>
    </StoreProvider>
  )
}
