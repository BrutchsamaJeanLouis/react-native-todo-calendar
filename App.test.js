import React from 'react'
import renderer from 'react-test-renderer'
import App from './App'

describe('<App />', () => {
  // it('has 1 child', () => {
  //   const tree = renderer.create(<AppBar />).toJSON()
  //   expect(tree.length).toBe(1) // The length of the tree should be three because we want a 3x3 grid
  // })
  it('renders correctly', () => {
    const tree = renderer.create(<App />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})