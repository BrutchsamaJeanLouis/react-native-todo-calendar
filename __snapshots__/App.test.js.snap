// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<View
  collapsable={false}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "height": "100%",
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <ActivityIndicator
      size="large"
    />
  </View>
</View>
`;
